server/src/cloud/user.ts

```diff
@@ -0,0 +1,31 @@
+import {requireValidBoardAdmin, requireValidBoardMember} from "./permission";
+import {api} from "./util";
+
+/**
+ * @param showHiddenColumns Allows moderators to toggle the display of hidden columns (visible only to themselves)
+ */
+type EditUserConfigurationRequest = {
+  userId: string;
+  boardId: string;
+  editUserConfigurationRequest: {
```

Comment:
I would name this one just `userConfiguration`, since you'll probably want to override all previous settings if new settings are defined and it's not just a request? The edit and request aspect itself is already defined by the type and the name of the endpoint so it seems redundant. Just my 2 cent on the naming

Reply:
Right. Thank you. But it will be also updated/refactored soon when #778 is done.